php_phongo_structs.h

```diff
@@ -21,15 +21,9 @@
 
 #include "php_bson.h"
 
-#if PHP_VERSION_ID >= 70000
 #define PHONGO_ZEND_OBJECT_PRE
 #define PHONGO_ZEND_OBJECT_POST zend_object std;
 #define PHONGO_STRUCT_ZVAL zval
```

Comment:
I don't think these macros need to exist anymore. Let's just replace them with the defined value. `PHONGO_ZEND_OBJECT_PRE` can be deleted entirely.